apps/core-app/src/renderer/src/modules/hooks/useUrlProcessor.ts

```
@@ -0,0 +1,72 @@
+export async function useUrlProcessor(): Promise<void> {
+  function directListener(event: Event): void {
+    const target = event.target as HTMLElement
+
+    if (target.nodeName.toLocaleLowerCase() === 'a') {
+      if (target.getAttribute('ignoreSafeCheck') === 'true') return
+
+      const url = target.getAttribute('href')
+
+      if (!url) return
+
+      const regex =
```

ESLint

[error] '=' should be placed at the beginning of the line.

(style/operator-linebreak)